docs/templates/.frogbot/frogbot-config.yml

```diff
@@ -13,6 +13,17 @@
       branches:
         - master
 
+      # [Optional] Template for the branch name generated by Frogbot when creating pull requests with fixes.The template must include ${BRANCH_NAME_HASH}, to ensure that the generated branch name is unique
+      branchNameTemplate: ""
```

Comment:
Let's put this line as a comment (with a # prefix) and set the default value as the value.
The same goes for the two templates below.